### PR TITLE
update links

### PR DESCRIPTION
**Pull request by Arhell**

### CODE REVIEW

File: src/Template/Pages/home.ctp [hunk 1]
```diff
@@ -205,15 +205,15 @@ $cakeDescription = 'CakePHP: the rapid development PHP framework';
                 <ul><li>Live chat about CakePHP</li></ul>
             </li>
             <li class="bullet cutlery">
-                <a href="http://cakesf.herokuapp.com/">Slack</a>
+                <a href="https://cakesf.herokuapp.com/">Slack</a>
```
Comment (garas) [inline src/Template/Pages/home.ctp]:
```suggestion
                <a href="https://slack-invite.cakephp.org/">Slack</a>
```

**Comment by othercorey** [comment]
I don't think we're updating 3.x anymore.

File: src/Template/Pages/home.ctp (hunk 1 above)
Comment (Arhell) [reply src/Template/Pages/home.ctp]: Done

**Comment by Arhell** [comment]
> I don't think we're updating 3.x anymore.

Removed the changes, leaving only the corrected Slack link

**Pull request merged**